src/perf/lib/ThroughputClient.cpp

```diff
@@ -579,9 +581,19 @@ ThroughputClient::StreamCallback(
         }
         MsQuic->StreamShutdown(StreamHandle, QUIC_STREAM_SHUTDOWN_FLAG_ABORT, 0);
         break;
-    case QUIC_STREAM_EVENT_SHUTDOWN_COMPLETE:
+    case QUIC_STREAM_EVENT_SHUTDOWN_COMPLETE: {
```

Comment:
Strictly speaking, you shouldn't need to add wrapping `{ ... }` here because everything is in the `if (PrintBlockedTimings) {` block.